include/tables/object_table.h

```diff
@@ -410,7 +410,7 @@
 /* 0x18D */ DEFINE_OBJECT(object_00F48000, OBJECT_397)
 /* 0x18E */ DEFINE_OBJECT(object_00FD6000, OBJECT_398)
 /* 0x18F */ DEFINE_OBJECT(object_00FD9000, OBJECT_399)
-/* 0x190 */ DEFINE_OBJECT(object_011F5000, OBJECT_400)
+/* 0x190 */ DEFINE_OBJECT(act_balloon, OBJECT_400)
```

Comment:
OBJECT_400 should be renamed to OBJECT_ACT_BALLOON. you will also have to rename it in the actor profile